thorvg/idf_component.yml

```
@@ -0,0 +1,8 @@
+version: "0.11.9"
+description: graphics library
```

```suggestion
description: "ThorVG is an open-source graphics library designed for creating vector-based scenes and animations"
```

we can use this sentence from ThorVG readme.